src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html

```diff
@@ -20,9 +20,9 @@
         <td class="col-4 text-break">{{ customer.id }}</td>
         <td class="col-4 text-break">{{ customer.name }}</td>
         <td class="col-2 text-center">
-          <button 
-            data-toggle="modal" 
-            (click)="editCustomer(customer.id)" 
+          <button
+            data-toggle="modal"
+            (click)="editCustomer(customer.id)"
```

Comment:
please see why you get these changes if it is the same.

Reply:
When we were trying to decide how to approach the problematic, in the beginning, we touched that part of the code, then it was re established, but perhaps with a space extra or without a space.